itest/testutil/app_cli.go

```diff
@@ -0,0 +1,127 @@
+package testutil
```

Comment:
nit: we should really be consistant in file naming, go convention is all concat, so in this case, `appcli.go`. Either we use snake case everywhere on be inline with go convention

Reply:
moved to `appcli.go`